### PR TITLE
Use pathlib in volumetrics. Move to rms test dir

### PR DESCRIPTION
**Pull request by berland**
The moving of test data is to align with #70

### CODE REVIEW

File: src/fmu/tools/rms/volumetrics.py [hunk 1]
```diff
@@ -55,7 +53,7 @@ def myregionrenamer(s):
     """
     # First find out which row the data starts at:
     headerline = 0  # 0 is the first line
-    with open(txtfile) as volfile:
+    with open(str(txtfile)) as volfile:
```
Comment (HansKallekleiv) [inline src/fmu/tools/rms/volumetrics.py:56]: Do we need to convert to `str`? `open` supports Path objects directly https://docs.python.org/3/library/os.html#os.PathLike

```suggestion
    with open(txtfile) as volfile:
```

File: tests/rms/test_rmsvolumetrics.py [hunk 2]
```diff
@@ -17,19 +11,14 @@
...
-        testdir = os.path.abspath(".")
 
-    volfiles = glob.glob(testdir + "/data/rmsvolumetrics/*txt")
+def test_volumetrics():
+    """Test parsing of many real examples from RMS"""
 
-    for file in volfiles:
-        dframe = volumetrics.rmsvolumetrics_txt2df(file)
+    for filename in TESTDIR.glob("*.txt"):
+        dframe = volumetrics.rmsvolumetrics_txt2df(str(filename))
```
Comment (HansKallekleiv) [inline tests/rms/test_rmsvolumetrics.py:21]:
```suggestion
        dframe = volumetrics.rmsvolumetrics_txt2df(filename)
```

File: tests/rms/test_rmsvolumetrics.py [hunk 3]
```diff
@@ -47,15 +36,15 @@ def myrenamer(a_zone):
...
+        TESTDIR / "14_geo_gas_1.txt", zonerenamer=myrenamer
     )
     assert "Larsson" not in dframe.to_string()
     assert "E3_1" in dframe.to_string()
 
     # Test columnrenamer:
     columnrenamer = {"Region index": "FAULTSEGMENT"}  # this will override
     dframe = volumetrics.rmsvolumetrics_txt2df(
-        testdir + "/data/rmsvolumetrics/" + "14_geo_gas_1.txt",
+        str(TESTDIR / "14_geo_gas_1.txt"),
```
Comment (HansKallekleiv) [inline tests/rms/test_rmsvolumetrics.py:47]:
```suggestion
        TESTDIR / "14_geo_gas_1.txt",
```

**Review by HansKallekleiv — CHANGES_REQUESTED** [review]

File: src/fmu/tools/rms/volumetrics.py (hunk 1 above)
Comment (jcrivenaes) [reply src/fmu/tools/rms/volumetrics.py:56]: Agree partially with Hans, but rather:
```
with txtfile.open() as volfile:
```

https://docs.python.org/3/library/pathlib.html#pathlib.Path.open

Comment (berland) [reply src/fmu/tools/rms/volumetrics.py]: `volumetrics.py` now also ported to `pathlib`

File: tests/rms/test_rmsvolumetrics.py (hunk 2 above)
Comment (berland) [reply tests/rms/test_rmsvolumetrics.py:21]: fixed

File: tests/rms/test_rmsvolumetrics.py (hunk 3 above)
Comment (berland) [reply tests/rms/test_rmsvolumetrics.py:47]: fixed``

**Review by HansKallekleiv — APPROVED** [review]
👍

**Pull request merged**